mypy/typeshed/stdlib/basedtyping.pyi

```diff
@@ -0,0 +1,3 @@
+from typing import _SpecialForm
+
+Untyped: _SpecialForm
```

Comment:
TODO: delete this and ship actual basedtyping instead

Reply:
https://github.com/KotlinIsland/basedtyping/pull/44

Reply:
I'll merge it with the fake stubs and update it before release.